python/pyspark/sql/readwriter.py

```diff
@@ -400,7 +400,8 @@ def json(
         Example 2: Read JSON from multiple files in a directory
 
         >>> import tempfile
-        >>> with tempfile.TemporaryDirectory() as d1, tempfile.TemporaryDirectory() as d2:
+        >>> with (tempfile.TemporaryDirectory(prefix="json2") as d1,
+        ...         tempfile.TemporaryDirectory(prefix="json3") as d2):
```

Comment:
IIRC, the syntax:

```py
with (..., ...):
    ...
```

is not supported in Python 3.8.

It should be without `()`.

```py
with ..., ...:
    ...
```